pkg/xmtp/message.go

```diff
@@ -35,6 +53,14 @@ func getContext(env *messageApi.Envelope) interfaces.MessageContext {
 				senderHmac = &parsed.SenderHmac
 			}
 		}
+	} else if messageType == topics.V3Conversation {
+		if message, err := parseGroupMessage(env.Message); err == nil {
+			*shouldPush = true
+			hmacInputs = &message.Data
+			if len(message.SenderHmac) > 0 {
+				senderHmac = &message.SenderHmac
+			}
```

Comment:
So the reason this is failing now is because the `message.SenderHmac` is 0

Reply:
That's so weird. It's definitely there on the lib side.